src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,117 @@
+.clock {
+  @include defaultCenter;
+  width: $clockSize;
+  height: $clockSize;
+  border-radius: $borderRadius;
+  border: 2px solid $mainColor;
+
+  &__number-3 {
+    position: absolute;
+    width: ($clockSize - 60);
+    color: $itemsColor;
+    transform: rotate(180deg);
+    @include selectChild(180deg);
+  }
+
+  &__number-6 {
+    position: absolute;
+    width: ($clockSize - 60);
+    color: $itemsColor;
+    transform: rotate(270deg);
+    @include selectChild(90deg);
+  }
+
+  &__number-9 {
+    position: absolute;
+    width: ($clockSize - 40);
+    color: $itemsColor;
+    transform: rotate(360deg);
+    @include selectChild(10deg);
+  }
+
+  &__number-12 {
+    position: absolute;
+    width: ($clockSize - 60);
+    color: $itemsColor;
+    transform: rotate(450deg);
+    @include selectChild(-90deg);
+  }
+
+  &__second {
+    @include defaultCenter;
+    position: absolute;
+    padding: $secondPadding;
+    border-radius: $borderRadius;
+    animation: timeSecondAnimation 60s infinite linear;
+    z-index: 2;
+
+    &-line {
+      position: absolute;
+      left: ($secondPadding * 2);
+      height: 1px;
+      width: ($clockSize / 2.6);
+      background: $mainColor;
+    }
+  }
+
+  &__minute {
+    @include defaultCenter;
+    position: absolute;
+    padding: $minutePadding;
+    border-radius: $borderRadius;
+    animation: timeMinuteAnimation 1440s infinite linear;
+    z-index: 1;
+
+    &-line {
```

Comment:
It is not a good idea to use scss nesting just not to write the same prefix again
The main purpose of nesting is to nest some related things like modifiers or pseudo-classes but not just the text prefix just because it is the same now